libs/sdk/tools/src/generators/update-dependencies/generator.spec.ts

```diff
@@ -183,15 +189,27 @@ describe('update dependencies generator', () => {
       directory: 'lib/test',
       skipFormat: true,
       skipPackageJson: true,
+      skipTests: true,
     });
     await libraryGenerator(appTree, {
-      name: 'other',
+      name: 'aaa',
       buildable: true,
       publishable: true,
-      importPath: '@proj/other',
-      directory: 'lib/other',
+      importPath: '@proj/aaa',
+      directory: 'lib/aaa',
       skipFormat: true,
       skipPackageJson: true,
+      skipTests: true,
+    });
+    await libraryGenerator(appTree, {
+      name: 'bbb',
+      buildable: true,
+      publishable: true,
+      importPath: '@proj/bbb',
+      directory: 'lib/bbb',
+      skipFormat: true,
+      skipPackageJson: true,
+      skipTests: true,
     });
```

Comment:
Adding coverage for the sort since the parent project no longer gets included in the implicit dependencies.